app/src/types/folders.ts

```diff
@@ -0,0 +1 @@
+export type Special = 'all' | 'mine' | 'recent';
```

Comment:
What does `Special` mean in this context? Could we name this something more specific to prevent future name collision?
Feels like it could be `FolderFilterType` or something similar 🤔

Reply:
It's for these special folder views:
<img width="209" alt="Screenshot 2023-05-01 at 15 18 43" src="https://user-images.githubusercontent.com/5363448/235457095-94972de7-e79a-488a-af4a-a3634564d072.png">

Probably `SpecialFolder`, `SpecialFolderView`? 🤔

Reply:
Both are more descriptive than just `Special` :)